osquery/tables/system/linux/md_tables.h

```diff
@@ -21,6 +21,10 @@ namespace tables {
 struct MDDrive {
   std::string name;
   size_t pos;
+
+  MDDrive() {
+    pos = 0;
```

Comment:
I'd not recommend this. There a 'modern` initialization recommendation:
```
size_t pos{0};
```
(assuming you edit line 23)

Or the existing more common:
```cpp
MDDrive() : pos(0) {}
```